docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -29,10 +29,10 @@ See GitHub documentation for instructions on how to clone repos.
 
 ## Set Default Template Location
 1. Open the `sfdx-config.json` config file in your `<project-folder>/.sfdx` folder
-2. Add the parameter `customOrgMetadataTemplates`, and set its value to either a local directory or your cloned GitHub repository that contains your default templates. The GitHub URL can either point to the root folder that contains your templates, or a path to a subfolder on a branch in the repo that contains your templates. For example:
+2. Add the parameter `customOrgMetadataTemplates`, and set its value to the templates folder in your cloned repo. For example:
 ```
 {
-"customOrgMetadataTemplates": "https://github.com/mygithubacct/salesforcedx-templates"
+  "customOrgMetadataTemplates": "https://github.com/mygitrepo/salesforcedx-templates/tree/main/packages/templates/src/templates"
```

Comment:
excellent!

Should we also include an example for pointing at a local copy of the cloned repo? 
`  "customOrgMetadataTemplates": "/Users/gbockus/github/PDT/salesforcedx-templates/packages/templates/src/templates"
` for instance?

Reply:
done @gbockus-sf